uiowa/uiowa
# Past Events

Resolves #3131 

Past events proof of concept utilizing existing block/paragraph type implementations. This requires an update to content.uiowa.edu. Still limited by the max 50 results.

## To Test
- ~~checkout `contenthub-past-events-instances` branch on **702 DEV**~~
- pull and sync a v2 and v3 website
- ~~`drush @[sitename].local config-set sitenow_events.settings base_endpoint https://content.dev.drupal.uiowa.edu/api/v1/`~~
- configure an events block/paragraph, see the "Display past events" option, save.
- see events starting yesterday and going backwards.

## To Merge

- [x] `contenthub-past-events-instances` has to be deployed to 703 PROD first
